source/vmware/vmware-overview.rst

```diff
@@ -0,0 +1,380 @@
+Kubernetes-based applications in production with all the power of virtualized
+storage, networking, and associated management. 
+
+Starting with VMware vSphere 7 Update 1 (7U1), vSphere includes MinIO's
+first-party integration through the vSAN Data Persistence Platform (DPP). 
+IT administrators using vSphere can provision high-performance multi-tenant
+object storage entirely through the vCenter interface and with minimal
+manual interaction with the underlying Kubernetes infrastructure. 
+
+IMAGE
+
+This documentation assumes familiarity with all referenced VMware components,
+including but not limited to VMware vCenter,VMware vSAN, VMware vSAN Data
+Persistence Platform, vSAN Direct Configuration, and vSAN SNA. While this
+documentation *may* provide guidance for configuring or deploying VMware
+components on a best-effort basis, it is not a replacement for dedicated
+VMware-specific documentation.
+
+vSAN Direct and vSAN SNA Storage
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+VMware vSphere 7U1 and the VMware Data Persistence Platform (DPP) offer
+two new vSAN data paths specifically designed for cloud-native stateful
+services like MinIO.
+
+.. class:: bold-dt
+
+vSAN SNA Drives
+   Allows creating vSAN storage objects that are co-located with their
+   associated compute resources. The MinIO tenant can utilize the local storage
+   with less overhead as compared to traditional vSAN networked storage.
+
+   vSAN SNA drives are ideal for clusters where the MinIO Tenant must share
+   the physical infrastructure with other stateful or stateless workloads.
+
+vSAN Direct Drives
+   Allows creating independent datastores on every disk attached to a node,
+   where applications running on that node can have direct access to those
+   disks. The MinIO Tenant can fully utilize the available throughput of the
+   drive with none of the overhead associated to distributed storage.
+
+   vSAN Direct drives are ideal for clusters dedicated to running the MinIO
+   Tenant.
+
+While MinIO can make efficient use of either data path, vSAN Direct most closely
+mirrors MinIO best practices around ensuring the shortest possible path between
+a MinIO process and its connected storage.
+
+MinIO recommends using only *one* type of vSAN data path for all drives intended
+for use by a given MinIO Tenant. vSphere administrators should create
+a :vmware7-docs:`Storage Policy
+<com.vmware.vsphere.storage.doc/GUID-720298C6-ED3A-4E80-87E8-076FFF02655A.html>`
+associated to either the vSAN SNA or vSAN Direct disks and ensure that the
+:vmware7-docs:`namespace
+<vmware-vsphere-with-tanzu/GUID-1544C9FE-0B23-434E-B823-C59EFC2F7309.html>` in
+which the MinIO Tenant will be deployed has access to that policy.
+
+.. ToDo: Link to vmware docs on configuring/provisioning Direct/SNA Drives
+
+.. _minio-vmware-enable:
+
+Enable MinIO for VMware vSphere
+-------------------------------
+
+VMware vSphere 7 Update 1 includes the MinIO Client Plugin. You have to
```

Comment:
Update 1 is already out.. we need to find the exact patch that will include MinIO
```suggestion
VMware vSphere 7 Update 1 XXX includes the MinIO Client Plugin. You have to
```